src/nvim/os/users.c

```diff
@@ -41,7 +41,11 @@ int os_get_usernames(garray_T *users)
 // Return OK if a name found.
 int os_get_user_name(char *s, size_t len)
 {
+#ifdef UNIX
   return os_get_uname(getuid(), s, len);
+#else
+  return os_get_uname(0, s, len);
```

Comment:
So is this a TODO, or is this a workable implementation for windows? If it's a TODO, I guess a `// TODO ...` comment could be useful.